iwyu.cc

```diff
@@ -2982,6 +2982,10 @@ class InstantiatedTemplateVisitor
     }
   }
 
+  void ReportDeclForwardDeclareUse(SourceLocation, const NamedDecl*,
+                                   const char* /*comment*/ = nullptr) override {
+  }
```

Comment:
Add a comment here to explain why we never want forward-declare uses of decls in template instantiation.